tests/common_setup.py

```diff
@@ -324,9 +345,10 @@ def enterprise_one_client_bootstrapped_with_gateway(request):
     new_tenant_client(
         env, "mender-client", tenant["tenant_token"], network="mender_local"
     )
+    env.device_group.ssh_is_opened()
+
     env.start_tenant_mender_gateway(tenant["tenant_token"])
     env.device_gateway = MenderDevice(env.get_mender_gateways(network="mender").pop())
-    env.device_group.ssh_is_opened()
```

Comment:
I am fine with this (it reads better) but the previous approach had a tiny performance improvement as you are waiting the QEMU boot time (until SSH) only once. Now you start the client(s), wait boot time, start the gateway, wait boot time.